changelogs/fragments/432-build-release-role-minor-refactoring.yml

```diff
@@ -0,0 +1,5 @@
+---
+breaking_changes:
+  - build-release role - the role now depends on community.general
+minor_changes:
+  - build-release role - preform minor refactoring of the build-release role, mostly concerning ``tasks/tests.yml``. This reduces use of ``shell`` and ``set_fact``, makes the role more robust, and replaces short names with FQCNs. (https://github.com/ansible-community/antsibull/pull/432)
```

Comment:
```suggestion
  - The ``build-release`` role now depends on the ``community.general`` collection (https://github.com/ansible-community/antsibull/pull/432).
minor_changes:
  - Perform minor refactoring of the ``build-release`` role, mostly concerning ``tasks/tests.yml``. This reduces use of ``shell`` and ``set_fact``, makes the role more robust, and replaces short names with FQCNs (https://github.com/ansible-community/antsibull/pull/432).
```

Reply:
> preform

This is what happens when I forget to turn on spell check in nvim :(. I've committed your changes.